src/component/CreateTeam.css

```diff
@@ -0,0 +1,18 @@
+.create-button-team {
+  background-color: #890aad; /* Button background color */
+  padding: 10px;
+  margin: 20px;
+  color: #fff; 
+  border: none; 
+  cursor: pointer;
+  border-radius: 5px;
+  text-align: center;
+  font-size: 15px;
+  width: 35%;
```

Comment:
Using percentages for layout widths can lead to unpredictable behavior on different devices. Please be aware of this or consider using a more stable approach like flexbox.